eventkit_cloud/tasks/task_runners.py

```diff
@@ -416,6 +419,15 @@ def create_format_task(format):
     CeleryExportTask = getattr(module, class_name)
     return CeleryExportTask
 
+def create_format_thematic_task(format):
+    task_fq_name = export_task_registry_thematic[format]
+    # instantiate the required class.
+    parts = task_fq_name.split('.')
+    module_path, class_name = '.'.join(parts[:-1]), parts[-1]
+    module = importlib.import_module(module_path)
+    CeleryExportTask = getattr(module, class_name)
+    return CeleryExportTask
+
```

Comment:
This seems really redundant to the create_format_task function, can we just pass the dict object in along with the format, `create_format_task(export_task_registry, format)`?